full-stack-tests/core/src/frontend/standalone/tile/TileRequest.test.ts

```diff
@@ -71,19 +67,18 @@ describe("IModelTileRequestChannels", () => {
       return getTileForIModel(imodel);
     }
 
-    it("is not configured by default", async () => {
-      expect(IModelApp.tileAdmin.channels.iModelChannels.cloudStorage).to.be.undefined;
+    it("is configured by default", async () => {
+      expect(IModelApp.tileAdmin.channels.iModelChannels.cloudStorage).not.to.be.undefined;
```

Comment:
If `IModelTileRequestChannels.cloudStorage` is now always enabled remove `| undefined` from its type declaration?

Reply:
Yeah good point.  I'll clean that up.

Reply:
pushed that change